src/main.rs

```diff
@@ -174,7 +176,7 @@ We would love to hear from you on the data APIs you need, please leave feedback
             .app_data(path_config)
             .wrap(actix_web::middleware::Logger::default())
             .wrap(prometheus.clone())
-            .app_data(web::Data::new(pool.clone()))
+            .app_data(web::Data::new(pool_explorer.clone()))
             .app_data(web::Data::new(db_helpers::DBWrapper {
```

Comment:
Is it possible to use the [newtype](https://doc.rust-lang.org/1.9.0/book/structs.html#tuple-structs) pattern here to strongly type both explorer/balance pools? It becomes a bit more verbose, but means we:
1. No longer need `DbWrapper` for passing two pools to `actix_web`
2. Can strongly type lower level functions so we don't mistakingly use the wrong pool

Type def:
```rs
struct ExplorerPool(sqlx::Pool<sqlx::Postgres>)
struct BalancesPool(sqlx::Pool<sqlx::Postgres>)
```
Passing to Actix web:
```rs
.app_data(web::Data::new(ExplorerPool(pool_explorer.clone())))
.app_data(web::Data::new(BalancesPool(pool_balances.clone())))
```
Then consuming:
```rs
pub(crate) async fn get_ft_history(
    ExplorerPool(pool_explorer): ExplorerPool,
    BalancesPool(pool_balances): BalancesPool,
```

I'm not sure if this will work, but it would be good to strongly type these somehow

Reply:
I like this, thank you!